{
    "assignee": null,
    "assignees": [],
    "body": "I 'm trying to use cython this way:\r\n`cython --embed -o betws.c betws.py`\r\n'betws.py' is my python file.\r\n\r\nBut I get this error (Expected an idenifier) on line 730:\r\n`query.exec(SQL)`\r\n\r\nThe whole code at this point is:\r\n`self.db.transaction()`\r\n`        for values in all_values:`\r\n`            SQL = \"INSERT INTO games (\" + fields + \") VALUES ('\" + \"','\".join(str(e) for e in values) + \"')\"`\r\n`            query.exec(SQL)`\r\n`        self.db.commit()`\r\n\r\n\r\nSQL is a string. How can I fix this?",
    "closed_at": "2017-06-10T09:37:50Z",
    "comment_data": [
        {
            "body": "I changed 'query.exec(SQL)' to 'query_.exec(SQL)' and seems ok!",
            "created_at": "2017-06-10T08:56:56Z",
            "html_url": "https://github.com/cython/cython/issues/1730#issuecomment-307552495",
            "id": 307552495,
            "issue_url": "https://api.github.com/repos/cython/cython/issues/1730",
            "updated_at": "2017-06-10T08:56:56Z",
            "url": "https://api.github.com/repos/cython/cython/issues/comments/307552495",
            "user": {
                "avatar_url": "https://avatars3.githubusercontent.com/u/6674193?v=3",
                "events_url": "https://api.github.com/users/panosss/events{/privacy}",
                "followers_url": "https://api.github.com/users/panosss/followers",
                "following_url": "https://api.github.com/users/panosss/following{/other_user}",
                "gists_url": "https://api.github.com/users/panosss/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/panosss",
                "id": 6674193,
                "login": "panosss",
                "organizations_url": "https://api.github.com/users/panosss/orgs",
                "received_events_url": "https://api.github.com/users/panosss/received_events",
                "repos_url": "https://api.github.com/users/panosss/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/panosss/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/panosss/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/panosss"
            }
        },
        {
            "body": "Yes, ``exec`` is a language keyword in Python 2.x (but no longer in Python 3.x).",
            "created_at": "2017-06-10T09:37:50Z",
            "html_url": "https://github.com/cython/cython/issues/1730#issuecomment-307554190",
            "id": 307554190,
            "issue_url": "https://api.github.com/repos/cython/cython/issues/1730",
            "updated_at": "2017-06-10T09:37:50Z",
            "url": "https://api.github.com/repos/cython/cython/issues/comments/307554190",
            "user": {
                "avatar_url": "https://avatars3.githubusercontent.com/u/491659?v=3",
                "events_url": "https://api.github.com/users/scoder/events{/privacy}",
                "followers_url": "https://api.github.com/users/scoder/followers",
                "following_url": "https://api.github.com/users/scoder/following{/other_user}",
                "gists_url": "https://api.github.com/users/scoder/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/scoder",
                "id": 491659,
                "login": "scoder",
                "organizations_url": "https://api.github.com/users/scoder/orgs",
                "received_events_url": "https://api.github.com/users/scoder/received_events",
                "repos_url": "https://api.github.com/users/scoder/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/scoder/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/scoder/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/scoder"
            }
        }
    ],
    "comments": 2,
    "comments_url": "https://api.github.com/repos/cython/cython/issues/1730/comments",
    "created_at": "2017-06-10T07:49:44Z",
    "event_data": [
        {
            "actor": {
                "avatar_url": "https://avatars3.githubusercontent.com/u/491659?v=3",
                "events_url": "https://api.github.com/users/scoder/events{/privacy}",
                "followers_url": "https://api.github.com/users/scoder/followers",
                "following_url": "https://api.github.com/users/scoder/following{/other_user}",
                "gists_url": "https://api.github.com/users/scoder/gists{/gist_id}",
                "gravatar_id": "",
                "html_url": "https://github.com/scoder",
                "id": 491659,
                "login": "scoder",
                "organizations_url": "https://api.github.com/users/scoder/orgs",
                "received_events_url": "https://api.github.com/users/scoder/received_events",
                "repos_url": "https://api.github.com/users/scoder/repos",
                "site_admin": false,
                "starred_url": "https://api.github.com/users/scoder/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/scoder/subscriptions",
                "type": "User",
                "url": "https://api.github.com/users/scoder"
            },
            "commit_id": null,
            "commit_url": null,
            "created_at": "2017-06-10T09:37:50Z",
            "event": "closed",
            "id": 1118195065,
            "url": "https://api.github.com/repos/cython/cython/issues/events/1118195065"
        }
    ],
    "events_url": "https://api.github.com/repos/cython/cython/issues/1730/events",
    "html_url": "https://github.com/cython/cython/issues/1730",
    "id": 234987528,
    "labels": [],
    "labels_url": "https://api.github.com/repos/cython/cython/issues/1730/labels{/name}",
    "locked": false,
    "milestone": null,
    "number": 1730,
    "repository_url": "https://api.github.com/repos/cython/cython",
    "state": "closed",
    "title": "Error: Expected an idenifier",
    "updated_at": "2017-06-10T09:37:50Z",
    "url": "https://api.github.com/repos/cython/cython/issues/1730",
    "user": {
        "avatar_url": "https://avatars3.githubusercontent.com/u/6674193?v=3",
        "events_url": "https://api.github.com/users/panosss/events{/privacy}",
        "followers_url": "https://api.github.com/users/panosss/followers",
        "following_url": "https://api.github.com/users/panosss/following{/other_user}",
        "gists_url": "https://api.github.com/users/panosss/gists{/gist_id}",
        "gravatar_id": "",
        "html_url": "https://github.com/panosss",
        "id": 6674193,
        "login": "panosss",
        "organizations_url": "https://api.github.com/users/panosss/orgs",
        "received_events_url": "https://api.github.com/users/panosss/received_events",
        "repos_url": "https://api.github.com/users/panosss/repos",
        "site_admin": false,
        "starred_url": "https://api.github.com/users/panosss/starred{/owner}{/repo}",
        "subscriptions_url": "https://api.github.com/users/panosss/subscriptions",
        "type": "User",
        "url": "https://api.github.com/users/panosss"
    }
}